data/human/news.txt

```diff
@@ -18,16 +18,16 @@ news "human merchant anywhere"
 			not attributes "human"
 	name
 		word
-			"Merchant captain"
```

Comment:
These drastically needed sorting, good call doing this it will make maintenance and additions much easier in the future!

Reply:
Yes, good change. But can we make the list-components actually sorted (alphabetically)? And can we define this list generically to avoid the duplication between the list here and "in human space".